### PR TITLE
Add QA steps page

### LINKED ISSUE #66: Document common QA steps to do when reviewing PRs
See [dev catch-up notes](https://github.com/ubuntudesign/meeting-notes/issues/279)

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Added a page on QA steps to go through, regardless of what the PR does (see [dev catch-up](https://github.com/ubuntudesign/meeting-notes/issues/279) about this). We could link this to a repo's PR template.

## QA

- Check that the QA steps are reasonable
- Add anything you think is important

## Issues

Fixes #66

### CODE REVIEW

**Comment by steverydz** [comment]
:+1:

**Comment by shivjoshi1996** [comment]
:+1:

File: workflow/qa-steps.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# QA steps
+
+In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
+
+## Adding content
+
+- If adding a new page to a site, check that it has been added to the navigation, sitemap and breadcrumbs (where applicable)
+- If applicable, check that the new content's copy matches the relevant copydoc.
+- Conversely, update the copydoc if it is out of date with the content.
```
Comment (nottrobin) [inline workflow/qa-steps.md]: I don't think this is the responsibility of the reviewer. "Check that the new content matches the relevant copydoc" should cover it - as in, let the person who did the work know of the discrepancy. Resolving the discrepancy is best done by the person doing the actual work.

**Comment by nottrobin** [comment]
@pmahnke [technically](https://github.com/canonical-webteam/practices/blob/master/CONTRIBUTING.md#reviewing-pull-requests), you're not supposed to use "request changes":

> The GitHub reviews features ("approve" and "request changes") should not be used, as all discussion is merely a discussion of opinions.

It may turn out to be a losing battle, but I just thought I'd mention it one more time.

File: workflow/qa-steps.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Removing a page
+
+- When removing a page, check that it has been removed from the navigation, sitemap and breadcrumbs (where applicable)
+- Ensure that redirects are in place for deprecated urls
```
Comment (nottrobin) [inline workflow/qa-steps.md]: :heart:

File: workflow/qa-steps.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- If the page you are QAing contains a form, fill it out and check that it still submits. Also, fill it out incorrectly and check that validation still works.
+
+## Browser testing
+
+- Check that the changes look correct on Chrome, Firefox and IE/Edge (see [Browser support](/coding/browser-support.md))
+
+## Responsiveness
+
+- Check that the changes look correct on mobile, tablet and desktop sizes.
```
Comment (nottrobin) [inline workflow/qa-steps.md]: Could this point we placed under "Browser testing"? It doesn't seem to quite warrant another additional heading.

File: workflow/qa-steps.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# QA steps
+
+In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
+
+## Adding content
+
+- If adding a new page to a site, check that it has been added to the navigation, sitemap and breadcrumbs (where applicable)
```
Comment (nottrobin) [inline workflow/qa-steps.md]: It's a tad pedantic, but you should probably be consistent about whether list items have full stops at the end or not.

File: workflow/qa-steps.md (hunk 3 above)
Comment (anthonydillon) [reply workflow/qa-steps.md]: I would defend the separate categorisation as they are different review aspects. For example, a reviewer should always check the different viewports on their default browser but it is not always necessary or reasonable to browser test all reviews.

**Comment by anthonydillon** [comment]
:+1:

**Pull request merged**